docs/roleplay/nation-leader-system.md

```diff
@@ -0,0 +1,28 @@
+# Nation Leader Responsibilities, Requirements, and Elections
+
+## Responsibilities of a Nation Leader
+
+A Nation Leader is required to actively lead and drive roleplay for their nation. They are role models for players to look up to for roleplay. More specific tasks include:
+
+- Collaborate with members of their nation to create and push exciting storylines
+- Organise, assist with and encourage casual roleplays and larger events
+- Communicate with other Nation Leaders and the playerbase to improve the current state of roleplay
+- Process roleplay applications
+
+## Requirements to Run for a Nation Leader Position
+
+- You must have a character introduction
+- Players who have seriously or repeatedly broken server rules must be approved by staff if they wish to run for a NL position
+
+## Elections
+
+1. Elections for a new Nation Leader will occur every four months, at the end of each roleplay year, for all four nations.
+[comment]: <> (Link to rp years thing)
+2. Candidates will put forward their character by creating a forum post in the [#elections](https://discord.com/channels/641386636453871656/1104875951378141314) channel in the [Roleplay Hub](https://discord.gg/tjqX25pH37). Candidates will campaign for one week and answer questions through this post. Voting will take place after this period.
+3. The Rokucraft Admin team will distribute a voting form in the roleplay announcement channels. The form will be open for three days. For each nation, players will be able to vote for all candidates that they favour, or to re-open nominations (RON). It will only allow players to fill out the form once and will ask players for their Minecraft username. After the three-day voting period is over, the results will be announced and duties will be transferred.
+4. If there are no candidates for a Nation Leader position, it will be left empty and the other Nation Leaders will manage OOC affairs.
+
+## Inactivity
+
+If a Nation Leader is unable to lead (e.g. due to personal circumstances), their council will inherit their duties until the next election. The council will handle character applications and lead the nation where necessary. If the Nation Leader returns at any point during their term, they will resume their duties. It will be at the discretion of the council to determine whether the Nation Leader is inactive or not. If the Nation Leader is deemed inactive, the council should notify them (e.g. ping in council chat or direct message).
```

Comment:
```suggestion
If a Nation Leader is unable to lead, such as due to personal circumstances, their council will inherit their duties until the next election. The council will handle character applications and lead the nation where necessary. If the Nation Leader returns at any point during their term, they will resume their duties. It will be at the discretion of the council to determine whether the Nation Leader is inactive or not. If the Nation Leader is deemed inactive, the council should notify them.
```